apps/src/code-studio/pd/application_dashboard/summary_table.jsx

```diff
@@ -16,6 +17,16 @@ const styles = {
   }
 };
 
+const status_colors = {
+  'unreviewed': color.charcoal,
+  'pending': color.lighter_orange,
+  'interview': color.orange,
+  'waitlisted': color.level_passed,
+  'accepted': color.level_perfect,
+  'declined': color.red,
+  'withdrawn': color.lightest_red
+};
```

Comment:
This seems redundant with the switch statement above (quick_view_table.jsx, line 85). Can this be moved into a shared module?

Reply:
agreed, will do.